aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -243,11 +238,11 @@ void f8f8bf16_rowwise_impl(
   using StrideOutput = typename Gemm::GemmKernel::StrideC;
 
   StrideInputA stride_a = cutlass::make_cute_packed_stride(
-      StrideInputA{}, cute::make_shape(M, K, 1));
+      StrideInputA{}, cute::make_shape(M, static_cast<int>(XQ.stride(0)), 1));
```

Comment:
Uhm, no, it's just saying that the innermost dimension (i.e., K) is contiguous. It would be equivalent to put `XQ.stride(1)` here, but since we have previously checked that this equals 1, I just hardcoded 1 right away.

Reply:
This is cutlass noob Q I guess I am suprrised that the strides appear to be of len 3 instead of 2, e.g. M, 1, versus (M, 1, 1

It seems the trailing dim is refers to a L (local tile) ?